terraform/deployments/tfc-configuration/variables-production.tf

```diff
@@ -307,7 +307,7 @@ module "variable-set-rds-production" {
         allocated_storage            = 1024
         instance_class               = "db.m6g.large"
         performance_insights_enabled = false
-        freestoragespace_threshold   = 536870912000
+        freestoragespace_threshold   = 716800000000
```

Comment:
It alerts on how much free space is left, so this would alert if there is less than ~700GB remaining (30% usage)

Reply:
Oops thanks. Fixed